mk/main.mk

```diff
@@ -263,7 +263,7 @@ endif
 ######################################################################
 
 # FIXME: x86-ism
-LLVM_COMPONENTS=x86 arm aarch64 mips ipo bitreader bitwriter linker asmparser mcjit \
+LLVM_COMPONENTS=x86 arm aarch64 mips powerpc ipo bitreader bitwriter linker asmparser mcjit \
```

Comment:
Due to this I think you'll also want to touch [the llvm-auto-clean-trigger](https://github.com/rust-lang/rust/blob/master/src/rustllvm/llvm-auto-clean-trigger) to ensure LLVM is rebuilt.